member_go.py

```
@@ -0,0 +1,105 @@
+from telethon.sync import TelegramClient
+from telethon.tl.functions.messages import GetDialogsRequest
+from telethon.tl.types import InputPeerEmpty
+import csv
+
+api_id = XXXXXX # Replace this with your own api_id
```

Ruff (0.8.2)

Undefined name `XXXXXX`

(F821)